editor/src/clj/editor/code/view.clj

```diff
@@ -62,7 +62,7 @@
            [javafx.scene.layout ColumnConstraints GridPane Pane Priority]
            [javafx.scene.paint Color LinearGradient Paint]
            [javafx.scene.shape Rectangle]
-           [javafx.scene.text Font FontSmoothingType TextAlignment]
+           [javafx.scene.text Font FontSmoothingType TextAlignment Text]
```

Comment:
Doesn't `Event` precede `EventHandler` and `Text` precede `TextAlignment` when you Organize Imports? 🤔

Reply:
I forgot to do it, you got me :D